tests/commands/test_onyo.py

```diff
@@ -10,27 +11,29 @@
 ]
 @pytest.mark.repo_dirs('just-a-dir')
 @pytest.mark.parametrize('variant', variants)
-def test_onyo_debug(repo, variant):
+def test_onyo_debug(repo: Repo, variant: str) -> None:
     ret = subprocess.run(['onyo', variant, 'mkdir', '--yes', f'flag{variant}'],
                          capture_output=True, text=True)
     assert ret.returncode == 0
     assert 'DEBUG:onyo' in ret.stderr
+    repo.fsck()
```

Comment:
Here I would suggest using:

```
from contextlib import nullcontext as does_not_raise

with does_not_raise():
    repo.fsck()
```

You're expecting this to pass without raising any exceptions, Since the function returns `None` and only raises an exception if there are problems, we can be specific about this in the tests.

You can read more about this [in the pytest docs](https://docs.pytest.org/en/6.2.x/example/parametrize.html#parametrizing-conditional-raising) on parametrizing conditional raising.